apps/E2E/src/TabsLegacy/pages/TabsLegacyPageObject.ts

```diff
@@ -25,53 +25,39 @@ class TabsLegacyPageObject extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
-  async waitForTabsItemsToOpen(tabItemSelector: TabItemSelector, timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.didTabItemContentLoad(tabItemSelector), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'The Tab Items content did not open.',
-      interval: 1000,
-    });
-  }
-
-  async getTabItemAccesibilityRole(tabItemSelector: TabItemSelector): Promise<string> {
-    return await (await this.getTabItem(tabItemSelector)).getAttribute('ControlType');
-  }
-
-  async clickOnTabItem(tabItemSelector: TabItemSelector): Promise<void> {
-    await (await this.getTabItem(tabItemSelector)).click();
+  async didTabItemContentLoad(tabItemSelector: TabItem): Promise<boolean> {
```

Comment:
Do we still need this function? Can we just consolidate this into `waitForTabItemContentToLoad()`?

Reply:
It's not needed.